themes/custom/droopler_theme/droopler_theme.theme

```diff
@@ -163,32 +163,52 @@ function droopler_theme_theme_suggestions_page_alter(array &$suggestions, array
 function droopler_theme_preprocess_input(&$variables) {
   // For submit elements.
   if ($variables['element']['#type'] == 'submit') {
+
     // If button has some class.
-    if (is_object($variables['attributes']['class'])) {
-      // If there is no .btn-{type} class.
-      if (empty(array_intersect([
-        'btn-primary',
-        'btn-secondary',
-        'btn-success',
-        'btn-warning',
-        'btn-danger',
-        'btn-link',
-      ], $variables['attributes']['class']->value()))
-      ) {
-        $variables['attributes']['class']->addClass('btn-primary');
-      }
-      // If there is no .btn class.
-      if (!in_array('btn', $variables['attributes']['class']->value())) {
-        $variables['attributes']['class']->addClass('btn');
-      }
+    if (is_object($variables['attributes'])) {
+      $classes = $variables['attributes']['class']->value();
     }
-    // If there are no classes at all.
     else {
-      $variables['attributes']['class'][] = 'btn';
-      $variables['attributes']['class'][] = 'btn-primary';
+      $classes = $variables['attributes']['class'] ?? [];
+    }
+
+    // Skip when no-bootstrap class is set.
+    if (in_array('no-bootstrap', $classes)) {
+      return;
+    }
+
+    // If there is no .btn class - add it.
+    if (!in_array('btn', $classes)) {
+      $classes[] = 'btn';
+    }
 
+    // If there is no .btn-{type} class.
+    if (empty(array_intersect([
+      'btn-primary',
+      'btn-secondary',
+      'btn-success',
+      'btn-warning',
+      'btn-danger',
+      'btn-link',
+    ], $classes))
```

Comment:
Wrzuciłabym to raczej do jakiejś arrayki, żeby w tym porównaniu uniknąć takiego piętrusa.